### ISSUE TITLE
Update Ansible to 2.9.2 version

### ISSUE THREAD

**Issue by php-coder** [php-coder/mystamps#1142]
The pinned and outdated version of Ansible brought to much headaches and it seems it's easier to update Ansible rather than continue to put more workarounds.

Also my initial assumption was that it's a regression and but it's turned out that it was actually a fix for buggy behaviour that we use: https://github.com/ansible/ansible/issues/29618

History:
- fc72d3b931903033c004cf0fad8dd849ab3816a5 pinning has been introduced
- 59f05efba34f44d146c07743980e8b10bb7551ca it broke our deploy (#1020)
- ef4abbbeb231adb5f9e92d24bdd6f3136fd5d0b0 it broke our deploy again (#1089) and I had to apply a workaround

@asm0dey JFYI

**Comment by php-coder** [comment]
P.S. And now it also conflicts with `ansible-lint` (see #1129) as it pulls the latest version of Ansible.

**Comment by php-coder** [comment]
One more point: in order to use `systemd` module (#998) we need Ansible 2.2+

**Comment by php-coder** [comment]
I think that we shouldn't unpin but just updated to the latest version. Having a fixed version is good for stable builds.

**Comment by php-coder** [comment]
New warnings after update to 2.2.30:
```
TASK [Stopping service] ********************************************************
 [WARNING]: raw module does not support the environment keyword
changed: [my-stamps.ru]

TASK [Copying WAR file] ********************************************************
changed: [my-stamps.ru]

TASK [Starting service] ********************************************************
 [WARNING]: raw module does not support the environment keyword
changed: [my-stamps.ru]
```
This was a bug (see https://github.com/ansible/ansible/issues/23764) that has been fixed in newer versions.

**Comment by php-coder** [comment]
```
TASK [Ensuring that WAR file exists] *******************************************
ok: [my-stamps.ru -> 127.0.0.1] => {
    "changed": false, 
    "msg": "All assertions passed"
}
```
It seems like we should try `quit` parameter to make the output cleaner.

**Comment by php-coder** [comment]
Warning on 2.9.0 when I run `ansible-playbook provisioning/prod.yml -i provisioning/prod.inventory --tags backup`:
```
TASK [mystamps-backup : Creating /data/backups] ************************************************************************************************************************************************************
[DEPRECATION WARNING]: Distribution Ubuntu 16.04 on host my-stamps.ru should use /usr/bin/python3, but is using /usr/bin/python for backward compatibility with prior Ansible releases. A future Ansible
release will default to using the discovered platform python for this host. See https://docs.ansible.com/ansible/2.9/reference_appendices/interpreter_discovery.html for more information. This feature
will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
ok: [my-stamps.ru]
```
and
```
TASK [mystamps-backup : Installing duplicity] **************************************************************************************************************************************************************
[WARNING]: Could not find aptitude. Using apt-get instead
```

**Comment by php-coder** [comment]
> It seems like we should try quiet parameter to make the output cleaner.

Done.

New warning after updating to 2.8.x:

> TASK [Stopping monitoring] *****************************************************
[DEPRECATION WARNING]: Distribution Ubuntu 16.04 on host my-stamps.ru should 
use /usr/bin/python3, but is using /usr/bin/python for backward compatibility 
with prior Ansible releases. A future Ansible release will default to using the
 discovered platform python for this host. See https://docs.ansible.com/ansible
/2.8/reference_appendices/interpreter_discovery.html for more information. This
 feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
ok: [my-stamps.ru]

See https://travis-ci.org/php-coder/mystamps/jobs/626895186

**Comment by php-coder** [comment]
> New warning after updating to 2.8.x:

I'll create an issue for that.

**Comment by php-coder** [comment]
Finally fixed in 1ebedcd commit.

/spent 9h36m
